sapi/wlan/rsi_wlan_apis.c

```diff
@@ -3214,9 +3214,19 @@ int32_t rsi_wlan_get(rsi_wlan_query_cmd_t cmd_type, uint8_t *response, uint16_t
   // If state is not in card ready received state
   if (common_cb->state == RSI_COMMON_STATE_NONE) {
     if (cmd_type == RSI_FW_VERSION) {
+#ifdef RSI_WITH_OS
+      uint32_t wait_count = 0;
+      const uint32_t wait_interval = 100; /* ms */
```

Comment:
change wait_interval to wait_interval_ms to indicate the interval is in milliseconds